### DISCUSSION TITLE
Engineering units

### DISCUSSION THREAD

**Discussion by adilbek-m** [equinor/neqsim#239] [Q&A] [answered]
Dear Prof. Solbraa,
I noticed your neqsim project in github. We are trying to use the library in our product. Thanks  for making such a wonderful project open source.I have a question with regards to engineering units of input parameters, in particular the following parameters used as inputs in pipeline function (in the parentheses my guesses):

- position (m)
- diameter (m)
- height (m)
- outTemp (oC)
- rough (m)
- outerHeatTransferCoefficients (W/(m2K))
- pipeWallHeatTransferCoefficients (W/(m2K))

Are my guesses correct? In general, are all engineering units in SI units of measure by default?

**Comment by EvenSol** [comment, marked as answer]
Thanks for your input.

Yes, all your guesses are correct 😊.
We try to stick to SI units in the development (one exception is that pressure is in unit bar in most equations).

You can have a look at the example in the notebook:
https://colab.research.google.com/github/EvenSol/NeqSim-Colab/blob/master/notebooks/fluidflow/gaspipeline.ipynb
